src/Hosting/Server.Abstractions/src/ServerIntegratedAuth.cs

```diff
@@ -1,21 +1,20 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-namespace Microsoft.AspNetCore.Hosting.Server
+namespace Microsoft.AspNetCore.Hosting.Server;
+
+/// <summary>
+/// Used by servers to advertise if they support integrated Windows authentication, if it's enabled, and it's scheme.
+/// </summary>
+public class ServerIntegratedAuth : IServerIntegratedAuth
 {
     /// <summary>
-    /// Used by servers to advertise if they support integrated Windows authentication, if it's enabled, and it's scheme.
+    /// Indicates if integrated Windows authentication is enabled for the current application instance.
     /// </summary>
-    public class ServerIntegratedAuth : IServerIntegratedAuth
-    {
-        /// <summary>
-        /// Indicates if integrated Windows authentication is enabled for the current application instance.
-        /// </summary>
-        public bool IsEnabled { get; set;}
+    public bool IsEnabled { get; set; }
```

Comment:
I just _know_ this was a test to see who would catch you slipping the space in here :smile:

Reply:
Found a half dozen or so new spaces. Should we be looking for more❔